sirepo/template/srw.py

```diff
@@ -1996,6 +2021,23 @@ def _load_user_model_list(model_name, qcall=None):
     return _load_user_model_list(model_name, qcall=qcall)
 
 
+def _machine_learning_percent_complete(run_dir, res):
+    dm = simulation_db.read_json(run_dir.join(template_common.INPUT_BASE_NAME)).models
+    count = len(
+        pkio.sorted_glob(
+            run_dir.join("ensemble").join("worker*").join("sim*").join("values.npy")
```

Comment:
> I think the fixed height on the element selector can be removed now that it isn't showing in a modal dialog. Otherwise it is a bit odd when it cuts off parts of the input fields. ![img3](https://user-images.githubusercontent.com/6516910/204928959-43a6e18f-768d-4a0f-bb7d-79b49f627f84.png)

I think it's still useful once you get more than a couple of elements. [78e56ad](https://github.com/radiasoft/sirepo/pull/5216/commits/78e56ad71e64ce99c96485f35e4534d2b6f62247) adds a border - does that help?

Reply:
I like the border, but I really think you don't need to fix the height to 360px because it won't always display well in different browser sizes. You could have the height be variable and set to fill to the remaining browser height, but that would probably be too much work (elegant does this on the Lattice panel - the Beamline Editor panel fills the full browser window).

For example - here there is space available to show more values:
![pic1](https://user-images.githubusercontent.com/6516910/205134078-0711d88b-2553-4252-9a44-c29ad18acda7.png)

And here there isn't and you get 2 scrollbars which is bad:
![pic2](https://user-images.githubusercontent.com/6516910/205134325-21faec1c-564f-473f-9e9e-3ed3f796d021.png)

Reply:
> join takes multiple args. It seems to me you just want to count values.npy with a walk tree, but I could be wrong. Hardwiring the directory structure here is going to make this rigid/fragile.

[e6ba5ef](https://github.com/radiasoft/sirepo/pull/5216/commits/e6ba5efeda1abec0d3acee9cb34b4a59c5b8d084)